cdf_fabric_replicator/extractor.py

```diff
@@ -317,6 +345,31 @@ def convert_lakehouse_data_to_df(self, file_path: str, token: str) -> DataFrame:
             )
             raise e
 
+    def convert_lakehouse_data_to_df_batch(
+        self, file_path: str, external_id: str, timestamp: str, token: str
+    ) -> Generator[DataFrame, None, None]:
+        try:
+            dt = DeltaTable(
+                file_path,
+                storage_options={"bearer_token": token, "user_fabric_endpoint": "true"},
+            )
```

Comment:
should this be use_fabric_endpoint? alternatively, did you look into the whole range of azure storage options (https://docs.rs/object_store/latest/object_store/azure/enum.AzureConfigKey.html#variants)?

Reply:
Yes it should, good catch.  Since it seems to have worked with the incorrect parameter, I'm wondering if this is even needed for lakehouse reads.  Will investigate

Reply:
Interestingly `use_fabric_endpoint` seems to be needed only for write, not read.  The bearer token is still required.  Looking at those storage options they look like different ways to auth to a storage account - there might be something there for MSI but in past investigations I haven't been able to configure the MSI endpoint.  I would push that to another investigation, we've been using the guidance [here](https://blog.fabric.microsoft.com/en-us/blog/delta-rs-introduces-native-support-for-onelake/) to auth to Fabric.